.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.pythonPath": "C:\\Users\\as\\miniconda3\\envs\\TF\\python.exe"
```

Comment:
This seems out right wrong we use a different default environment now, but it's just in the settings and can be changed later just keep it mind.